src/envoy/http/mixer/report_data.h

```diff
@@ -84,6 +90,19 @@ class ReportData : public ::istio::control::http::ReportData {
     }
     return false;
   }
+
+  bool GetGrpcStatus(GrpcStatus *status) const override {
```

Comment:
IIRC, Sometimes when response body is empty, then trailers_ will be empty too. In this case grpc status and message are sent in response headers.

Reply:
Thanks.  updated the code to get grpc-status from both trailers and response headers.